Sources/Core/iOS/ActivityIndicatorViewObserver.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ActivityIndicatorViewObserver.swift
+//  Operations
+//
+//  Created by Matthew Holden 16/04/2016.
```

Comment:
I wasn't sure whether putting my name here was desired/OK or not.   On WebKit they ask you to add your name as a new "©" entry if you "make significant changes" to the file.

I'm fine either way :)